docs/install/docker/README.md

```diff
@@ -61,6 +61,39 @@ You also need to update the `VIRTUAL_HOST` entry in the `docker-compose.yml` fil
 
 For more details on the options available, see the [configuration guide](../configuration.md).
 
+### MQTT Broker
+
+FlowForge includes a MQTT broker to allow projects/devices to comunicate with core platform. This will be enabled by default, 
+you can dissable it by commenting out the following lines in the `docker-compose.yml` file
```

Comment:
Do we really want to spell out how to disable it?

The localfs instructions say it is optional as an eass-of-use thing for quickly trying out the platform. I'm not sure we want to encourage disabling it in the other environments.

Reply:
OK, just wanted to be clear that it's optional